backend/src/main/java/com/bakdata/conquery/models/jobs/CalculateCBlocksJob.java

```diff
@@ -32,9 +32,17 @@
 import lombok.Setter;
 import lombok.extern.slf4j.Slf4j;
 
-@RequiredArgsConstructor @Slf4j
+/**
+ * Calculate CBlocks, ie the Connection between a Concept and a Bucket.
+ * <p>
+ * If a Bucket x Connector has a CBlock, the ConceptNode will rely on that to iterate events. If not, it will fall back onto equality checks.
```

Comment:
Some doku sugar

```suggestion
 * If a {@link Bucket} x {@link Connector} has a {@link CBlock}, the {@link ConceptNode} will rely on that to iterate events. If not, it will fall back onto equality checks.
```

Reply:
Kann man dann gegebenenfalls resourcen freigeben, wenn für ein konzept ein CBlock generiert werden konnte. Also resourcen die sonst für ein `fall back onto equality checks` gebracuht würden?
Vielleicht wird das auch schon gemacht, aber ich hab es nicht gesehen.

Reply:
> Also resourcen die sonst für ein fall back onto equality checks gebracuht würden?

Bei den equality checks werden eben keine ressourcen verwendet, das ist dann der case in ConceptNode#98